pom.xml

```diff
@@ -99,7 +109,11 @@
         <version>${configuration-as-code.version}</version>
         <scope>test</scope>
     </dependency>
-
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20240303</version>
+    </dependency>
```

Comment:
https://plugins.jenkins.io/json-api/

Reply:
Unfortunately, it seems that we'd need a higher version of Jenkins (2.404.1) to use this plugin. The existing plugin works fine as-is, so I don't think that we need to change it for now. Also, refer to [this](https://github.com/jenkinsci/datadog-plugin/pull/404#discussion_r1555711413).